### PR TITLE
[new release] jose (0.10.0)

### PR DESCRIPTION
**Pull request by ulrikstrid**
JOSE implementation for OCaml and ReasonML

- Project page: <a href="https://ulrikstrid.github.io/ocaml-jose">https://ulrikstrid.github.io/ocaml-jose</a>
- Documentation: <a href="https://ulrikstrid.github.io/ocaml-jose">https://ulrikstrid.github.io/ocaml-jose</a>

##### CHANGES:

- Upgrade mirage-crypto and remove cstruct (by @anmonteiro, special thanks to @hannesm for help with debugging a unexpected test failure)

### CODE REVIEW

File: packages/jose/jose.0.10.0/opam [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/ulrikstrid/reason-jose.git"
+url {
+  src:
+    "https://github.com/ulrikstrid/ocaml-jose/releases/download/v0.10.0/jose-0.10.0.tbz"
+  checksum: [
+    "sha256=a3559d2dd6fd3a8aa03f8f65b1af508a26475dcfe1448175e109b9b9924cc0f2"
+    "sha512=d31462f6d8194579c1774fdaa9ca7c0f0a8f5fb47ce87cc77e28a09ce43a921163113f0a991b8f576674111336b230a534e22573fd3f8fee133513f25632e65e"
```
[the remark this answers is not in the document]
Comment (ulrikstrid) [reply packages/jose/jose.0.10.0/opam:54]: I had issues with dune-release so I tried doing it by hand, I'll try to link up with someone on slack to maybe re-do this release.

**Comment by ulrikstrid** [comment]
The tests here seems strange that they would break on only ARM, either this is a bug in ptime (which we use for comparing the times) or something else is broken. Can we just disable the tests for ARM?

**Comment by ulrikstrid** [comment]
Hopefully I did this correctly

**Comment by mseri** [comment]
Thanks

**Pull request merged**